src/main/java/net/imagej/ops/filter/FilterNamespace.java

```diff
@@ -133,6 +133,28 @@
 		return result;
 	}
 
+	/**
+	 * Executes a bilateral filter on the given arguments.
+	 * 
+	 * @param in
+	 * @param out
+	 * @param m_SigmaR
+	 * @param m_SigmaS
+	 * @param m_radius
```

Comment:
Is there a special reason you use `m_` prefix in the names? I've seen this in KNIME for `private` variables, but nowhere in the ImageJ universe, so I'd assume this doesn't conform to imagej-ops code style. Also, I'd prefer simple names such as `sigmaR`, `sigmaS` and `radius`.